src/pushover_complete/pushover_api.py

```diff
@@ -1,10 +1,16 @@
+import io
+import os
+
 try:
-    from urllib.parse import urljoin
+    from pathlib import Path
```

Comment:
@cthoyt What was your thought on try/except imports? Where should they be located?

Reply:
Six doesn't have pathlib and also doesn't have changes between `file` and the new `_io` which is the real problem